tasks/yum.yml

```diff
@@ -27,9 +27,7 @@
     dest: '{{ item }}'
     regexp: '^\s*gpgcheck: 0'
     replace: 'gpgcheck: 1'
-  with_flattened:
-    - '/etc/yum.conf'
-    - '{{ yum_repos.stdout_lines| default([]) }}'
+  with_flattened: "{{ ['/etc/yum.conf', yum_repos.stdout_lines | default([]) ] }}"
```

Comment:
I don't know about this one...
What's the advantage here? Right now, all it does for me is make it harder to parse..

Reply:
It removes ansible-lint errors related to bare variables. Do you have any other ideas how to fix this error?

Reply:
It can be "fixed" like this:
```  with_flattened:
    - ['/etc/yum.conf']
    - '{{ yum_repos.stdout_lines| default([]) }}'
```
However I'm not sure if this is not maybe a bug with ansible-lint. I'll ask there.

Reply:
See: https://github.com/willthames/ansible-lint/issues/315

Reply:
Ok, we will see how it goes, because I was also surprised by this error, and consider this a dirty fix.

Reply:
To go on with this PR: How about you keep the task the way it is in master and add a `skip_ansible_lint` tag to it? And add a comment to reference the willthames/ansible-lint#315 issue?